config/default.yml

```diff
@@ -1169,6 +1169,7 @@ RSpec/Rails/MinitestAssertions:
   Description: Check if using Minitest matchers.
   Enabled: pending
   VersionAdded: '2.17'
+  VersionChanged: "<<next>>"
```

Comment:
This change should not have been made. This should only be changed when the default settings are changed. For example, when you add a configurable option or change the default for a configurable option.

Reply:
fix: https://github.com/rubocop/rubocop-rspec/pull/1777

Reply:
Oops, my bad. I should have caught that in review.

Thanks for discovering, and fixing.

Reply:
for what it's worth I feel like that was required by `bundle exec rake`, but it might have been I just messed something up or changed something else I shouldn't have locally to make it think this needed updating 🤷 

I'll keep this in mind for future contributions and let you know if I do find the tasks behaving wrongly

Reply:
Never mind! Thank you for your work.